code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -1,281 +1,271 @@
+/**
+ * # Facehugger
+ *
+ * A simplemob used by aliens to reproduce.  Will attempt to impregnate humans on its own, but can also be picked up and thrown for more efficiency.
+ *
+ * A small, crab-like creature which latches onto the faces of humans and monkeys and impregnates them with an alien embryo.  After a time, the embryo
+ * will explode the "parent" and an alien larva will be born.  Facehuggers can be deterred by using face-covering helmets or impossible to remove masks.
+ * They will generally bash against living and synthetic beings they cannot impregnate, eventually killing them.  Aliens can pick them up while alive and
+ * throw them, while humans or other handy creatures can only do so while they are dead.
+ */
+/mob/living/simple_animal/hostile/facehugger
+	name = "alien"
+	desc = "It has some sort of a tube at the end of its tail."
+	icon = 'icons/mob/alien.dmi'
+	icon_state = "facehugger"
+	icon_living = "facehugger"
+	icon_dead = "facehugger_dead"
+	gender = NEUTER
+	health = 10
+	maxHealth = 10
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	obj_damage = 0
+	attack_verb_continuous = "flails at"
+	attack_verb_simple = "flail at"
+	attack_sound = 'sound/weapons/bladeslice.ogg'
+	faction = list(ROLE_ALIEN)
```

Comment:
do humans with xenomorph organs end up getting the alien faction?

Reply:
No, but the facehuggers do a check on their target as to whether it has a baby inside of it.  If it does, the facehuggers just sort of crowd around them and do nothing, which is actually pretty cool if you ask me.  It's like they're waiting in anticipation.